ghost/i18n/locales/de/portal.json

```diff
@@ -1,5 +1,5 @@
 {
-    "(save {{highestYearlyDiscount}}%)": "",
+    "(save {{highestYearlyDiscount}}%)": "(jetzt sparen {{highestYearlyDiscount}}%)",
```

Comment:
@ronaldlangeveld should be `(jetzt {{highestYearlyDiscount}}% sparen)`

("save now 20%" vs. "save 20% now")

(Sorry for jumping into this merged PR, just found it in an open tab and thought I'd share my two native speaker cents.)